cmd/release.go

```diff
@@ -257,5 +259,8 @@ func getChangelog(version string, rc io.ReadCloser) (string, string) {
 		}
 	}
 
-	return header, strings.Join(s, "\n")
+	if releaseHeader == "" {
+		return "", "", errors.New("unable to locate release information in changelog for selected version '" + projInfo.Version + "'.")
```

Comment:
well technically `releaseHeaderPattern` would be nice to know maybe here in this case as well? What if someone will put `#` instead of `##` or will drop space between `##` and version? minor nit (:

Reply:
sorry for late suggestion!

Reply:
Fixed!